integration/query_test.go

```diff
@@ -37,81 +37,133 @@ func TestQueryBadFindType(t *testing.T) {
 	ctx, collection := s.Ctx, s.Collection
 
 	for name, tc := range map[string]struct {
-		value any
-		err   string
+		value any // optional, used for find value
+
+		err        *mongo.CommandError // required
+		altMessage string              // optional, alternative error message
+		skip       string              // optional, skip test with a specified reason
 	}{
 		"Document": {
 			value: bson.D{},
-			err:   "object",
+			err: &mongo.CommandError{
+				Code:    2,
+				Name:    "BadValue",
+				Message: "collection name has invalid type object",
+			},
+			altMessage: "collection name has invalid type object",
```

Comment:
This `altMessage` is identical to the error message.

```suggestion
```

Reply:
Also for the same reason https://github.com/FerretDB/FerretDB/pull/2806/files#r1222350133